docs/kubernetes/ingress-usage.md

```diff
@@ -240,6 +240,26 @@ a unique value. Read more about this in our
 
      flowId("reuse")
 
+# Filters - return fast
+
+Sometimes you just want to return a header, redirect or even static
+html content. You can return from skipper without doing a proxy call
+to a backend, if you end your filter chain with `<shunt>`.
+
+## Return static content
+
+To set a response header `X: bar`, return
```

Comment:
I'd prefer is this sentence started more like "The following example sets a response..."